src/main/java/com/metamx/common/io/smoosh/SmooshedFileMapper.java

```diff
@@ -131,10 +132,20 @@ public ByteBuffer mapFile(String name) throws IOException
   }
 
   @Override
-  public void close() throws IOException
+  public void close()
   {
+    Throwable thrown = null;
     for (MappedByteBuffer mappedByteBuffer : buffersList) {
-      ByteBufferUtils.unmap(mappedByteBuffer);
+      try {
+        ByteBufferUtils.unmap(mappedByteBuffer);
+      } catch (Throwable t) {
```

Comment:
FYI, there are a few `Throwable`s that, in general, should probably not allow continuation in the running process.

http://www.scala-lang.org/api/2.10.6/index.html#scala.util.control.NonFatal$ has some good examples of things which should probably not continue. 

Digging through the call stack for `unmap` though, it doesn't look like it throws anything that should be caught.